packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -114,6 +114,7 @@ describe('internal-plugin-metrics', () => {
       sinon.useFakeTimers(now.getTime());
       cd = new CallDiagnosticMetrics({}, {parent: webex});
       sinon.stub(uuid, 'v4').returns('my-fake-id');
+      cd.setDeviceInfo(webex.internal.device);
```

Comment:
This is existing code but, I feel the variable name 'cd' does not seem to follow proper naming conventions. Consider using a more descriptive name that reflects its purpose, such as 'callDiagnostics' or 'callMetrics', to improve code readability and maintainability. 
@mkesavan13, what do you think ?

Reply:
As much as I agree, I think it will be a huge change set in the PR. Will take it as a separate item after this PR gets merged. Should that be okay?

Reply:
@mkesavan13 Sure, no issues.